components/wso2is7.key.manager/src/main/java/org/wso2/is7/client/model/TenantModel.java

```diff
@@ -0,0 +1,265 @@
+        private String lastname;
+
+        @SerializedName("provisioningMethod")
+        private String provisioningMethod = "inline-password";
+
+        @SerializedName("additionalClaims")
+        private List<AdditionalClaims> additionalClaims = null;
+
+        public Owner(String username, String password, String email, String firstname, String lastname,
+                     String provisioningMethod, List<AdditionalClaims> additionalClaims) {
+            this.username = username;
+            this.password = password;
+            this.email = email;
+            this.firstname = firstname;
+            this.lastname = lastname;
+            this.provisioningMethod = provisioningMethod != null ? provisioningMethod : "inline-password";
+            this.additionalClaims = additionalClaims;
+        }
+
+        public String getUsername() {
+            return username;
+        }
+
+        public void setUsername(String username) {
+            this.username = username;
+        }
+
+        public String getPassword() {
+            return password;
+        }
+
+        public void setPassword(String password) {
+            this.password = password;
+        }
+
+        public String getEmail() {
+            return email;
+        }
+
+        public void setEmail(String email) {
+            this.email = email;
+        }
+
+        public String getFirstname() {
+            return firstname;
+        }
+
+        public void setFirstname(String firstname) {
+            this.firstname = firstname;
+        }
+
+        public String getLastname() {
+            return lastname;
+        }
+
+        public void setLastname(String lastname) {
+            this.lastname = lastname;
+        }
+
+        public String getProvisioningMethod() {
+            return provisioningMethod;
+        }
+
+        public void setProvisioningMethod(String provisioningMethod) {
+            this.provisioningMethod = provisioningMethod;
+        }
+
+        public List<AdditionalClaims> getAdditionalClaims() {
+            return additionalClaims;
+        }
+
+        public void setAdditionalClaims(List<AdditionalClaims> additionalClaims) {
+            this.additionalClaims = additionalClaims;
+        }
+    }
+
+    /**
+     * Represents additional claims for the tenant owner.
+     */
+    public static class AdditionalClaims {
+        @SerializedName("claim")
+        private String claim;
+
+        @SerializedName("value")
+        private String value;
+
+        public String getClaim() {
+            return claim;
+        }
+
+        public void setClaim(String claim) {
+            this.claim = claim;
+        }
+
+        public String getValue() {
+            return value;
+        }
+
+        public void setValue(String value) {
+            this.value = value;
+        }
+    }
+
+    /**
+     * Represents the request payload to update an owner of a tenant.
+     */
+    public static class OwnerPutModel {
```

Comment:
when naming model objects avoid using the operation name.  eg: OwnerRequest

Reply:
Shall we move the nested classes as individual classes which can be seperated from the parent class